app/models/project.rb

```diff
@@ -34,6 +36,13 @@ def self.search(search, page)
       .paginate(per_page: 5, page: page)
   end
 
+  def self.search_by_language(search, page)
+    includes(:languages)
+      .where("languages.name ILIKE ?", "%#{search}%")
```

Comment:
I think this partial match thing means that when we search for Java that we also match JavaScript - I think it might make more sense to just match the language name exactly ...

Reply:
It took me a some research and playing around to get what you meant by partial match... so the % is a wildcard that matches partial words, as you pointed out. This is something I noticed that bothered me, but I hadn't fixed it. I have removed them and I think it works now.